managers/oclcClassify.py

```diff
@@ -67,6 +67,27 @@ def getClassifyResponse(self):
 
         return self.parseXMLResponse()
 
+    def generate_identifier_query(self):
+        identifier_map = { "isbn": "bn",
+                          "issn": "in",
+                          "oclc": "no"}
+        return f"{identifier_map[self.identifierType]}: {self.identifier}"
+
+    def generate_author_title_query(self):
+        return f"ti:{self.title} au:{self.author}"
+
+    def generate_search_query(self):
+        """Parses the received data and generates an OCLC search query based either
+        on an identifier (preferred) or an author/title combination.
+        """
+        if self.identifier and self.identifierType:
+            return self.generate_identifier_query()
+        elif self.title and self.author:
+            return self.generate_author_title_query()
+        else:
+            print('Cannot search for work without identifier or title/author')
```

Comment:
The function I am adapting has a print statement like this, but I do wonder how much value it adds - maybe it would be better to call a logger and pass some information from the record? Maybe this is best done when refactoring the manager as a whole.

Reply:
I think let's remove and ensure the function that handles the ClassifyError is properly logging the error. 

I think the error message log should include the record id or uuid so we can reference it.

Reply:
I'm not 100% sure the ClassifyManager retains a reference to the record at this point because at this point I believe one record could have fired off potentially several different requests to Classify/OtherEditions based on the identifiers present. (For example, while we can find which edition was created from which record, I don't think we have a record of which DCDW record passed an OCLC number to rabbitmq to grab the OCLC single catalog record... but I could be wrong/hope I'm wrong! I removed the print statement and I'll double check that assumption.